CADD.pm

```diff
@@ -115,9 +115,10 @@ sub new {
 
   my $params = $self->params_to_hash();
   my @files;
+  $self->{non_sv_ann_file} = 0;
   # Check files in arguments
   if (!keys %$params) {
-    warn "WARNING: Using snv or indels CADD annotation file with structural variant can increase run time exponentially.".
+    warn "WARNING: Using snv or indels CADD annotation file with structural variant can increase run time exponentially. ".
```

Comment:
Suggesting that it has the same warning here as in L132 for uniformity so should be 

warn "WARNING: Using snv and/or indels CADD annotation file with structural variant can increase run time exponentially.".
         "Consider creating separate input files for SNV/indels and SV and use appropriate CADD annotation file.\n"

Reply:
Done!